cmd/installer/cli/root.go

```diff
@@ -100,6 +100,7 @@ func RootCmd(ctx context.Context, name string) *cobra.Command {
 	cmd.AddCommand(JoinCmd(ctx, name))
 	cmd.AddCommand(ShellCmd(ctx, name))
 	cmd.AddCommand(NodeCmd(ctx, name))
+	cmd.AddCommand(EnableHACmd(ctx, name))
```

Comment:
@ajp-io should this be nested under root?

Reply:
i don't see a place that makes sense other than `install`, but I don't know that i like that either. i'm open to it though.

Reply:
but then it inherits all the command line flags from `install`, and it doesn't need those